src/locales/en/translation.json

```diff
@@ -199,6 +199,24 @@
         }
       }
     },
+    "pagePersistIdentity": {
+      "title": "You have already proved your identity",
+      "header": "You have already proved your identity",
+      "content": {
+        "paragraph1": "You proved your identity when you used your GOV.UK account before. You will\nnot need to do it again to use this service.",
```

Comment:
should we delete the \n newline content? Don’t think it's adding anything (there's on on line 208 as well)